content/encrypt-extra.md

```diff
@@ -18,17 +18,17 @@ tableOfContents: true
 
 The <u>Disks</u> application can be used to encrypt the extra drive and it comes pre-installed on Pop!_OS and Ubuntu.
 
-In this screenshot the extra drive can be seen and then select the **Gear icon** under Volumes. Then select the **Format Partition...** option to open the 'Format Volume' window.
+In this screenshot, the extra drive can be seen. Select the **Gear icon** under Volumes. Then select the **Format Partition...** option to open the 'Format Partition' dialog.
 
 ![Disks](/images/encrypt-extra/Disk-Menu.png)
 
-Then use the 'Format Volume' window to format the volume with the options to name the volume and use the Ext4 partition with LUKS.
+When the 'Format Volume' dialog window opens, set the volume name and select the `Ext4` type with `LUKS`. Click on `Next` at the top right of the dialog, then click on format.
 
 ![Disks](/images/encrypt-extra/Disk-Encrypt.png)
 
 ## Decrypt on Login
 
-If we would like to decrypt the drive on login into the system there are settings for the encryption. In the <u>Disks</u> application on the drive that is encrypted then click on the LUKS partition and then the **Gear icons** on the bottom of the Volumes table. Then click on **Edit Encryption Options...** then unclick **User Session Defaults**. Now enter the Passphrase for the encrypted drive and hit **OK** button. It will ask for your password and once it is entered the drive will be decrypted on login.
+If we would like to decrypt the drive on login into the system there are settings for the encryption. In the <u>Disks</u> application, select the drive that is encrypted then click on the LUKS partition. Click on the **Gear icons** on the bottom of the Volumes table, then click on **Edit Encryption Options...**. Unclick **User Session Defaults**. Now enter the Passphrase for the encrypted drive and hit **OK** button. It will ask for your password and once entered, the drive will be decrypted on login.
```

Comment:
>`and hit **OK** button.`

I know this started out this way before your edits, but I'm wondering if this is a good opportunity to add a "the" to change this to `and hit the **OK** button.`.

Reply:
Sure